src/main/java/com/backend/webproject/controller/AdminSideController.java

```diff
@@ -31,8 +32,9 @@ public String showCouponsPage(Model model) {
     }
 
     @RequestMapping("/validateCoupon/{couponCode}")
-    @ResponseBody
```

Comment:
Why was the ResponseBody annotation erased?

Reply:
My mistake, I did not noticed it was erased when fixing merge problems

Reply:
Ohh okok, just put it back, I think it is necesary.

Reply:
I have already make the correction